.github/workflows/slack-notify.yml

```
@@ -39,21 +45,153 @@ jobs:
+            fi
+
+            if [ "${{ steps.readiness.outputs.has_failing_tests }}" = "true" ]; then
+              if [ -n "$details" ]; then
+                details="$details "
+              fi
+              details="${details}Failing checks: ${{ steps.readiness.outputs.failing_checks }}."
+            fi
+
+            if [ -n "$details" ]; then
+              text="$text
+$details"
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)